admin-ui/src/pages/settings-page.tsx

```
@@ -0,0 +1,880 @@
+  return (
+    <Card className="gap-4 py-4">
+      <CardHeader className="px-4">
+        <CardTitle>General</CardTitle>
+        <CardDescription className="hidden sm:block">
+          Default model routing and API key storage.
+        </CardDescription>
+      </CardHeader>
+      <CardContent className="space-y-4 px-4">
+        <div className="grid gap-2">
+          <Label className="text-muted-foreground text-xs">{smallModelLabel}</Label>
+          {hasModels ? (
```

ESLint

[error] Delete `·(`

(prettier/prettier)